### ISSUE TITLE
Included objects data not in correct format

### ISSUE THREAD

**Issue by pmcconnell** [oddnetworks/oddworks#44]
When a request is made for an object along with an includes query the format of the include field is not in standard json format. It is missing the attributes field. Some of the data normally in the attributes field is present in the root of the include but not nested.

In addition to probably being nonconforming to the json spec it makes parsing the data in the SDKs more work.

Sample response below from fetching nasa homepage view

```json
{
  "data": {
    "id": "homepage",
    "type": "view",
    "relationships": {
      "promotion": {
        "data": {
          "id": "daily-show",
          "type": "promotion"
        }
      },
      "featuredMedia": {
        "data": {
          "id": "0db5528d4c3c7ae4d5f24cce1c9fae51",
          "type": "video"
        }
      },
      "featuredCollections": {
        "data": {
          "id": "ab2d92ee98b6309299e92024a487d4c0",
          "type": "collection"
        }
      }
    },
    "attributes": {
      "channel": "nasa",
      "title": "Nasa Sample Homepage"
    },
    "links": {
      "self": "http://localhost:3000/views/homepage"
    },
    "meta": {}
  },
  "links": {
    "self": "http://localhost:3000/views/homepage?include=featuredMedia,featuredCollections"
  },
  "meta": {
    "channel": "nasa",
    "platform": "APPLE_IOS"
  },
  "included": [
    {
      "id": "0db5528d4c3c7ae4d5f24cce1c9fae51",
      "type": "video",
      "title": "Trek to Mount Sharp Begins",
      "description": "After busily exploring the Glenelg region of Gale Crater, Curiosity is moving on. The rover is starting a 5 mile, or about an 8 kilometer trek southwest towards the foothills of Mt Sharp.",
      "images": {
        "aspect16x9": "https://spaceholder.cc/1920x1080"
      },
      "url": "http://www.nasa.gov/sites/default/files/com20130711-320-jpl.mp4",
      "duration": 1234567,
      "meta": {
        "source": "itunes"
      },
      "channel": "nasa"
    },
    {
      "id": "ab2d92ee98b6309299e92024a487d4c0",
      "type": "collection",
      "title": "NASACast: What's Up? Video Podcasts",
      "images": {
        "aspect16x9": "https://spaceholder.cc/1920x1080"
      },
      "relationships": {
        "entities": {
          "data": [
            {
              "id": "b99ab89d33c654277b739dadc53a2822",
              "type": "video"
            },
            {
              "id": "9d788294f5f4e3e24d39b88e350237a4",
              "type": "video"
            },
            {
              "id": "73004256c4cb062ae228d8912f1aa388",
              "type": "video"
            },
            {
              "id": "b2892c0f3411a16dcc3fce93e1d36c66",
              "type": "video"
            },
            {
              "id": "b931f8353efbc720d0e8eef61e391a57",
              "type": "video"
            },
            {
              "id": "943af21ce037461b77c1752073c0a2a1",
              "type": "video"
            }
          ]
        }
      },
      "meta": {
        "source": "itunes"
      },
      "channel": "nasa"
    }
  ]
}

**Comment by brkattk** [comment]
This is a bug @pmcconnell - good find

**Comment by blainsmith** [comment]
This should be fixed by @pmcconnell from a previously merged PR this morning.

**Comment by brkattk** [comment]
Fixed by #47
